blocks/logos/logos.js

```diff
@@ -300,23 +300,4 @@ export default function decorate(block) {
       slider.init();
     }
   });
-  const slider = new CarouselSlider(block, {
```

Comment:
has this something to do with search?

Reply:
no, but It is some code I forgot there and passed a previous PR